global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -128,72 +150,94 @@ data:
-          severity: critical
-          region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+          severity: critical | warning | info
+          region: staging | lab-1
 
-      - receiver: slack_by_tier_and_severity
+      # ---- special notification channels ----
+      # owner: Thomas Grainchen
+      - receiver: slack_nannies
         continue: true
         match_re:
-          tier: os|kks|metal|net
-          severity: info|warning
-          region: admin|qa-de-1|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+          context: nanny 
 
-      # limit tier k8s to controlplane for now
-      # should be merged with the above block as soon as we sorted out kube-monitoring
-      - receiver: slack_by_tier_and_severity
+      # owner: Michael Schmidt / Fabian Ruff
+      - receiver: slack_k8s
         continue: true
         match_re:
           tier: k8s
+          severity: critical|warning|info
           cluster_type: controlplane
-          severity: info|warning
           region: admin|qa-de-1|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+      
+      # owner: Michael Schmidt / Fabian Ruff
+      # critical channel has the action buttons
+      - receiver: slack_kks_default
+        continue: true
+        match_re:
+          tier: kks
+          severity: warning|info
+          region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+      
+      - receiver: slack_kks_critical
+        continue: true
+        match_re:
+          tier: kks
+          severity: critical
+          region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
 
-      - receiver: slack_by_tier_and_service
+      # owner: Fabian Ruff
+      - receiver: slack_concourse
         continue: true
         match_re:
-          tier: os
           severity: info|warning|critical
-          service: arc|backup|barbican|cinder|cfm|designate|elektra|elk|hermes|ironic|keystone|limes|lyra|maia|manila|neutron|nova|sentry|swift
+          service: concourse
 
-      - receiver: slack_by_tier
+      # owner: Tilo Geissler
+      - receiver: pagerduty_alertchain_test
+        continue: false
+        match_re:
+          tier: test-tier
+          severity: test
+          region: area51
+
+      # owner: Tilo Geissler
+      - receiver: slack_by_os_service
         continue: true
         match_re:
+          tier: os
           severity: info|warning|critical
-          service: concourse
+          service: arc|backup|barbican|cinder|cfm|designate|elektra|elk|glance|hermes|ironic|keystone|limes|lyra|maia|manila|neutron|nova|sentry|swift
 
       # do not page for kubernikus controlplane
       - receiver: dev-null
```

Comment:
Since you specified the `cluster_type` on pagerduty receivers to not be ` kubernikus-controlplane|kubernikus-scaleout` this receiver can be removed.

Reply:
so what shall we sue the dev-null for then?

Reply:
Receivers are processed in the order they are configured. The dev-null receiver is meant to catch everything else -  meaning at least `region: staging`
There are 2 options:
(1) Move dev-null receiver to the top and use it to filter alerts we don't want to see anywhere else, like `region: staging, cluster_type: ...`. 
(2) Leave dev-null receiver at bottom but without any `match_re` filter. Must ensure alerts that are routed before to their final receiver are not getting here `continue: false`. Bonus: This would enable us to see misconfigured alerts (missing labels) in the dev-null channel.

Reply:
option (2) is the better one in my eyes .. going for that - so dev-null has a defined purpose, too //👍